cloud/disk_manager/test/acceptance/test_runner/README.md

```diff
@@ -1,25 +1,61 @@
 # Acceptance test runner
 
-## Configuration
-
-- `verify-test` can be built from sources [verify_test_sources](https://a.yandex-team.ru/arc/trunk/arcadia/cloud/blockstore/tools/testing/verify-test).
-- `acceptance-test` can be built from sources [acceptance_test_sources](https://a.yandex-team.ru/arc/trunk/arcadia/cloud/disk_manager/test/acceptance).
-
-1. Install `ycp` and build `acceptance-test` (see [setup_instruction](https://a.yandex-team.ru/arc/trunk/arcadia/cloud/disk_manager/test/acceptance)).
-2. Build `verify-test`:
-    ```(bash)
-    $ ya make ../../../../blockstore/tools/testing/verify-test/
-    ```
-3. Build `test-runner`:
-    ```(bash)
-    $ ya make
-    ```
-
-## "acceptance" test type
+These acceptance tests are intended to perform acceptance testing of disk-manager 
+and NBS as a part of the whole cloud. They are intended to run on a separate machine 
+with access to lab/production cluster and with internal tool `ycp` installed.
+These tests can't be launched locally.
+
+## Terminology
+
+* `Disk` - either a non-replicated disk with mirroring capabilities (basically SSD cut into chunks and shared via RDMA) or kikimr blob storage based disks. Blob storage based disks can be overlay (reference other disks).
```

Comment:
Lets not add to much details about implementation. 
Non-replicated disks can also be overlay by the way